rimage/image_file.go

```diff
@@ -22,6 +24,55 @@ import (
 	ut "go.viam.com/rdk/utils"
 )
 
+// RGBABitmapMagicNumber represents the magic number for our custom header
+// for raw RGBA data. The header is composed of this magic number followed by
+// a 4-bit line of the width as a uint32 number and another for the height. Credit to
```

Comment:
I think you mean 4-byte.  4 bytes = 32 bits

Reply:
I did